packages/bp/src/orchestrator/studio-client.ts

```diff
@@ -111,6 +111,7 @@ export const startStudio = async (logger: sdk.Logger, params: StudioParams) => {
     APP_SECRET: params.APP_SECRET,
     ROOT_PATH: params.ROOT_PATH,
     NLU_ENDPOINT: params.NLU_ENDPOINT,
+    CLOUD_NLU_ENDPOINT: params.NLU_ENDPOINT,
     SERVER_ID: process.SERVER_ID,
```

Comment:
@franklevasseur we're adding this because we always need to set it when testing the cdm locally. Any objection to this?

Reply:
nope

Ideally, if you really want to replicate whats hosted, you would set `NLU_ENDPOINT` to a nlu server on branch `next` and `CLOUD_NLU_ENDPOINT` to a server on branch `cloud`, but this will work